doctrine/doctrine-bundle/1.12/config/packages/doctrine.yaml

```diff
@@ -6,6 +6,9 @@ doctrine:
         # either here or in the DATABASE_URL env var (see .env file)
         #server_version: '5.7'
 
+        # backtrace queries in profiler (increases memory usage per request)
+        #profiling_collect_backtrace: '%kernel.debug%'
```

Comment:
another note: I'd suggest putting this after the charset options, as server_version+charset are related and should stay as a group to me

Reply:
i grouped by commented nodes :sweat_smile: better like this :+1: